### DISCUSSION TITLE
Fusion assembly and part files

### DISCUSSION THREAD

**Discussion by tinducvo** [fjp/2wd-robot#70] [Q&A]
Could we get the remo robot fusion 360 files? I want to use an Orange 5, rpLidar A1, and maybe a larger battery. It'd be easier with the parametric files as opposed to STLs. Thank you.

**Comment by fjp** [comment]
Hi @parkournerd, thanks for the interest. So far there hasn't been too much public interest, that I know of, for the Fusion 360 files. So I haven't provided them anywhere yet. I am still thinking about a fair way to offer the Fusion 360 files. It took quite some time and effort to adapt the Jetbot design, and all the work on REMO was done in collaboration with a designer. So I hope you understand this is not something I've currently planned to share freely.

However, there is already the RPLidar A1 available, which was [kindly contributed by another user](https://github.com/ros-mobile-robots/remo_description/issues/2#issuecomment-968876821). I just uploaded the stl that was shared with me (please see the linked post), to the branch [feature/rplidar-a1](https://github.com/ros-mobile-robots/remo_description/blob/feature/rplidar-a1/meshes/remo/lidar_platform/platfom_rplidar_a1.stl).

The swappable deck for the Orange 5 should be pretty straight forward to model even with the STLs, but I agree that the battery compartment height could be increased and the Fusion 360 files could be helpful for this.

I hope that helps for now. Please let me know if you are still interested in the Fusion 360 files and I will try to come up with a sharing strategy that is hopefully fair for everyone. Beside the first option below ;-) is there something else you would prefer?

1. Should be completely free (with Share-alike or GPLv3.0, ... license)
2. Accessible through GitHub Sponsors
3. Gumroad Access or similar site (still need to figure out a fair price)
4. Share with contributors
5. ... other ideas?

I am absolutely willing to provide the Fusion 360 files, so the project can be even more useful for others. A feedback from you or others about this would definitely help me to take the next steps. Thank you!

**Reply by tinducvo** [reply]
There has been a resounding amount of work on this project, so it makes sense you'd seek a good compromise:
- Compensation: monetary and/or attribution
- Control of the source: STL is more open

I'd probably consider where you want to take the project. I see three options:
1. Freemium OSS like you're doing, sharing mechanical files with contributors => less elegant to contribute, revenue like now
2. FOSS with more prominent donation => more contributors makes the project more robust project with donation potential. Are people more likely to pay immediately or donate once they benefit? I don't know xD
3. Get NVIDIA sponsorship so donations are unnecessary => may be difficult
